package.json

```diff
@@ -109,16 +104,16 @@
     "jest": "^24.1.0",
     "lerna": "^3.10.7",
     "lint-staged": "^9.0.0",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
```

Comment:
Do you need to bump `lodash` version as part of this upgrade?

Reply:
Yes, there are packages using `17.11`, `17.14` and `17.15`. I updated and "deduped" it